torchbenchmark/util/extra_args.py

```diff
@@ -146,3 +148,16 @@ def apply_opt_args(model: 'torchbenchmark.util.model.BenchmarkModel', args: argp
         module, exmaple_inputs = model.get_module()
         precision = 'fp16' if not model.dargs.precision == "fp32" else 'fp32'
         model.set_module(enable_torchtrt(precision=precision, model=module, example_inputs=exmaple_inputs))
+
+    if args.optimize_dynamo_ddp:
```

Comment:
Just curious what kind of optimizations does torchdynamo do for ddp optimizations?
Also, to prevent code bloating in this file, how about we move this part in `torchdynamo.py`?

Reply:
https://github.com/pytorch/torchdynamo/pull/628 adds extra graph breaks in dynamo. So instead of DDP having to wait until the entire backward pass is completed, extra graph breaks should allow autograd hooks to get called earlier, and then you can get better overlap of communication (syncing the gradients once they are ready) and computation (computing the rest of the backward pass)